src/components/organisms/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback } from "react";
```

Comment:
Can you please move this component to `src/pages`.

Reply:
Had the same idea late yesterday. Will move it and merge

Reply:
moved @ https://github.com/sparkletown/sparkle/pull/2366